src/modules/practice-client-intakes/routes.config.ts

```
@@ -1,7 +1,26 @@
+import { rateLimit } from '@/shared/middleware/rateLimit';
+import {
+  attachIntakeOwnership,
+  authorizeIntakeOwnership,
+} from '@/modules/practice-client-intakes/middleware/authorize-intake';
```

ESLint

[error] `@/modules/practice-client-intakes/middleware/authorize-intake` import should occur before import of `@/shared/middleware/rateLimit`

(import/order)